src/w3c/templates/headers.js

```diff
@@ -74,6 +74,17 @@ const localizationStrings = {
     latest_published_version: "Versión publicada mas reciente:",
     this_version: "Ésta versión:",
   },
+  de: {
+    author: "Autor/in:",
+    authors: "Autor/innen:",
```

Comment:
I don't know German, but are those `/` supposed to be there?

Reply:
In German there is no gender-neutral term for "author". We don't know the gender of the author, so it is common practice to use `/` to make clear both genders are included (since `Autor` is male and `Autorin` is female).

Reply:
Interesting. Thanks for clarification!